firefox-ios/Client/Nimbus/NimbusFeatureFlagLayer.swift

```diff
@@ -47,6 +47,9 @@ final class NimbusFeatureFlagLayer {
         case .feltPrivacySimplifiedUI:
             return checkFeltPrivacyFeature(for: featureID, from: nimbus)
 
+        case .feltPrivacyFeltDeletion:
```

Comment:
Seeing as we're using the same function, why not add this to the above case?

Reply:
updated!